cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -790,6 +809,67 @@ public void testResources() {
         assertThat(ccContainer.getResources().getRequests(), is(requests));
     }
 
+    @ParallelTest
+    public void testCpuCapacityGeneration() {
```

Comment:
This seems to cover the overrides well. I would be good to have also coverage for these scenario:
* Resource requests not set at all in brokers
* Resource requests have only a limit in the brokers

Reply:
Extra testing added